system-variables.md

```diff
@@ -791,27 +831,30 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 
 - Scope: SESSION | GLOBAL
 - Default value: 1024
-- Minimum value: 32
+- Range: [32, 2147483647]
 - This variable is used to set the maximum number of rows in a chunk during the execution process. Setting to too large of a value may cause cache locality issues.
 
 ### tidb_max_delta_schema_count <span class="version-mark">New in v2.1.18 and v3.0.5</span>
 
 - Scope: GLOBAL
 - Default value: 1024
+- Range: [100, 16384]
 - This variable is used to set the maximum number of schema versions (the table IDs modified for corresponding versions) allowed to be cached. The value range is 100 ~ 16384.
 
 ### tidb_mem_quota_apply_cache <span class="version-mark">New in v5.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: 32 MB
-- This variable is used to set the memory usage threshold of the local cache in the `Apply` operator.
+- Default value: 33554432
```

Comment:
Yes I agree. I have added units for the variables where I know them. It is not specifically tracked internally, so it will not be available on all variables.